exercises/E-conditionals/exercise.js

```diff
@@ -8,6 +8,9 @@
 
 let name = "Daniel";
 let danielsRole = "mentor";
+if (danielsRole === "mentor") {
```

Comment:
```suggestion
if (danielsRole === "mentor") {
console.log("Hi, I'm " + name + ", I'm a " + danielsRole + ".");
}
else if (danielsRole === "student") {
  console.log("Hi, I'm " + name + ", I'm a " + danielsRole + ".");
}
```

Reply:
nice work mesgna, there i find one more way to do this and this is my suggested review part of our coursework :smiley: